guides/common/modules/proc_creating-image-only-hosts.adoc

```diff
@@ -10,8 +10,9 @@ In {Project}, you can use {CRname} provisioning to create hosts from an existing
 . Click the *Organization* and *Location* tabs to ensure that the provisioning context is automatically set to the current context.
 . From the *Host Group* list, select the host group that you want to use to populate the form.
 . From the *Deploy on* list, select the {CRname} connection.
-. From the *Compute Profile* list, select a profile to use to automatically populate virtual machine settings.
```

Comment:
Good catch. I meant to revisit and confirm this step as I didn't see this step/field in the beta env UI. thanks for confirming though, that makes sense (and it didn't show in my UI because the profile wasn't created) i'll re-add the step.